lib/active_fedora/rdf/field_map_entry.rb

```diff
@@ -15,14 +16,15 @@ def initialize
     # Merges any existing values for solr fields with new, incoming values and ensures that resulting values are unique.
     # @param [Symbol] type the data type for the field such as :string, :date, :integer
     # @param [Array] behaviors Solrizer's behaviors for indexing such as :stored_searhable, :symbol
-    # @param [Array] values existing values for the solr field
-    def merge!(type, behaviors, values)
+    # @param [Array] new_values existing values for the solr field
```

Comment:
Does the comment also need to be updated? Is there an even better name for this argument? (`new_values_to_add` or something like that?)